README.tfdoc.hcl

```diff
@@ -0,0 +1,1223 @@
+                Time period, in seconds, during which requests from a client
+                should be routed to the same target group. The range is
+                `1`-`604800` seconds (7 days).
+              END
+            }
+
+            attribute "enabled" {
+              type        = bool
+              default     = false
+              description = <<-END
+                Whether target group stickiness is enabled.
+              END
+            }
+          }
+        }
+
+        attribute "redirect" {
+          type        = object(redirect)
+          description = <<-END
+            Configuration block for creating a redirect action. Required if
+            `type` is `redirect`.
+          END
+
+          attribute "status_code" {
+            type        = string
+            default     = "HTTP_302"
+            description = <<-END
+              HTTP redirect code. The redirect is either permanent (`HTTP_301`)
+              or temporary (`HTTP_302`).
+            END
+          }
+
+          attribute "host" {
+            type        = string
+            default     = "HTTP_302"
+            description = <<-END
+              Hostname. This component is not percent-encoded. The hostname can
+              contain `#{host}`.
+            END
+          }
+
+          attribute "path" {
+            type        = string
+            default     = "#{host}"
+            description = <<-END
+              Absolute path, starting with the leading `"/"`. This component is
+              not percent-encoded. The path can contain `#{host}`, `#{path}`,
+              and `#{port}`.
+            END
+          }
+
+          attribute "port" {
+            type        = string
+            default     = "#{port}"
+            description = <<-END
+              The port. Specify a value from `1` to `65535` or `#{port}`.
+            END
+          }
+
+          attribute "protocol" {
+            type        = string
+            default     = "#{protocol}"
+            description = <<-END
+              The protocol. Valid values are `HTTP`, `HTTPS`, or `#{protocol}`. 
+            END
+          }
+
+          attribute "query" {
+            type        = string
+            default     = "#{query"
+            description = <<-END
+              The query parameters, URL-encoded when necessary, but not
+              percent-encoded. Do not include the leading "?".
+            END
+          }
+        }
+      }
+
+      variable "tags" {
+        type        = map(string)
+        default     = {}
+        description = <<-END
+          A map of tags to apply to the `aws_lb_listener` resource.
+        END
+      }
+    }
+
+    section {
+      title = "Extended Resource Configuration"
+
+      variable "rules" {
+        type        = list(rule)
+        default     = []
+        description = <<-END
+          A list of rules to be attached to the created listener.
+        END
+
```

Comment:
ID attribute seems to be missing

Reply:
[**https://github.com/mineiros-io/terraform-aws-lb-listener/blob/main/variables.tf#L181**](https://github.com/mineiros-io/terraform-aws-lb-listener/blob/main/variables.tf#L181)